tests/unit/test_app.py

```diff
@@ -4,37 +4,36 @@
 """The unit tests for the  flask app."""
 
 import json
+import logging
 import secrets
-from pathlib import Path
+import sys
 from typing import Callable, Iterator
 
 import pytest
 from flask import Flask
 from flask.testing import FlaskClient
 from werkzeug.exceptions import BadRequest, UnsupportedMediaType
 
-import src.app as app_module
+import webhook_router.app as app_module
 from tests.unit.helpers import create_correct_signature, create_incorrect_signature
 
 TEST_PATH = "/webhook"
 
 
-@pytest.fixture(name="webhook_logs")
-def webhook_logs_fixture(tmp_path: Path):
-    """Return a path for the webhook logs."""
-    return tmp_path / "webhook.log"
-
-
 @pytest.fixture(name="app")
-def app_fixture(webhook_logs: Path) -> Iterator[Flask]:
-    """Setup the flask app."""
+def app_fixture() -> Iterator[Flask]:
+    """Setup the flask app.
+
+    Setup testing mode and add a stream handler to the logger.
+    """
     app_module.app.config.update(
         {
             "TESTING": True,
         }
     )
```

Comment:
I think this is not used in the flask application?

Reply:
This is used for testing, see https://flask.palletsprojects.com/en/3.0.x/config/#TESTING

Reply:
Learned, thanks!